src/libraries/Microsoft.PowerFx.Connectors/PowerPlatformConnectorClient.cs

```diff
@@ -51,12 +52,13 @@ public PowerPlatformConnectorClient(string endpoint, string environmentId, strin
             // Must set to allow callers to invoke SendAsync() via other helper methods.
             BaseAddress = new Uri("https://" + endpoint); // Uri.Parse will validate endpoint syntax. 
         }
-                
+
+        [SuppressMessage("Reliability", "CA2000: Dispose objects before losing scope", Justification = "Callers must manage response properly")]
```

Comment:
>objects [](http://example.com/codeflow?start=57&length=7)

which object is it complaining about?  Is there actually a missing dispose in here?

Reply:
This test isn't super clever I think.
It identifies we create a disposable object (HttpRequestMessage) that isn't disposed. 
This is ok but expected in this case as we return it!...
The callers will have to deal with this object and call Dispose when necessary.
Hence the need to suppress that message here.